docs/getting_started/tutorial.md

```diff
@@ -119,15 +119,15 @@ When logged into the admin site, a superuser has full permissions and is able to
 python manage.py runserver
 ```
 
-If everything worked, <http://127.0.0.1:8000> will show you a welcome page:
+After running the above command,the server generates a URL <http://127.0.0.1:8000>. Copy and paste the link into your browser tab, which displays the welcome page.
```

Comment:
I do not think we need this change, hopefully someone doing this tutorial knows how URLs and a browser work. 

We should avoid aiming this page too low, otherwise it will become too wordy for those that do know how a browser, URLs and the terminal works.

Reply:
Thanks for the feedback